### ISSUE TITLE
Handling of field that contains multiple lines

### ISSUE THREAD

**Issue by case-inlite** [janisdd/vscode-edit-csv#116]
### Description
The CSV editor does not properly handle fields that contain multiple lines. This results in the editor either not recognizing the file as a valid CSV and not displaying an "Edit CSV" button, or rendering a blank screen when the button is pressed. =

### Expected behavior
The CSV editor should recognize a field wrapped with double quotes that contains multiple lines as a single field in a row and render it correctly. Additionally, the enclosing double quotes should be preserved when saving the file.

### Steps to reproduce
1. Create a CSV file with the following content:
```
Name,Age,Address
Alice,30,"123 Main St,
New York, NY"
Bob,25,"456 Elm St,
Los Angeles, CA"
```
2. Open the CSV file in the CSV editor.
3. Observe the following possible behaviors:
   a. The editor does not recognize the file as a valid CSV.
   b. The "Edit CSV" button is not displayed.
   c. The "Edit CSV" button is displayed, but a blank screen appears when the button is pressed.

**Comment by janisdd** [comment]
Can't reproduce this. I opened a new file, pasted your example switched language (bottom right) to `CSV` and clicked the `Edit csv` button -> works.

What language is displayed in the bottom right status bar?

**Comment by case-inlite** [comment]
Disabling Rainbow CSV solved my problem, thanks for pointing me in the right direction. Apologies for taking up your time!

**Comment by case-inlite** [comment]
@janisdd I reenabled Rainbow Edit and while it plays nice with some CSVs, others trigger Dynamic CSV. I have been unable to figure out why. 

When I try and change the format to CSV it doesn't seem to stick, but changing it to CSV (Semicolon) does. Can't share current CSVs as the data is sensitive.

CSV0.7.4 and Rainbow v3.6.0.

**Comment by janisdd** [comment]
@case-inlite I think this is now fixed in version 0.7.5. The "command" was working but the button was just not displayed.

**Comment by case-inlite** [comment]
@janisdd thanks seems to be working reliably now
